config/queryparam-extensions.json

```diff
@@ -22,30 +22,13 @@
       "type": "string",
       "propertyDef": {
         "min": 0,
-        "max": "1",
+        "max": "*",
```

Comment:
The FHIR spec for family HumanName only allows for a single surname to be specified, its only the given name that allows for multiple values. We can revert this to have a max of **1**
https://www.hl7.org/fhir/datatypes.html#HumanName

Reply:
Doh, thanks for double checking all this for me.

Reply:
Cool no problem :)